app/Helpers/render/comment.php

```diff
@@ -113,7 +114,7 @@ function RenderArticleComment(
     $deleteIcon = '';
 
     if ($user && $user === $localUser || $allowDelete) {
-        $img = "<img src='" . asset('assets/images/icon/cross.png') . "' width='16' height='16' alt='delete comment'/>";
+        $img = Blade::render('<x-fas-xmark class="text-red-700 h-5 w-5" alt="delete comment"/>');
```

Comment:
nit: Recommend bumping up to `red-600` for improved contrast across color light/dark/black color schemes.